tests/unit/cli/test-fixtures/full.yaml

```diff
@@ -30,6 +30,11 @@ providers:
     request_timeout: 20
     releases:
       jinx: 87
+  epss:
+    runtime: *runtime
+    request_timeout: 20
+    dataset: yesterday
```

Comment:
I think the valid options are `current` and a date in `"YYYY-MM-DD"` format. Can we update this test fixture to have a valid config?